README.md

```diff
@@ -568,29 +587,26 @@ fun getSingingCertificates(packageName: String): Set<String>? {
 Concerning this redirection we have two cases that depend on whether the
 method ``startActivityForResult()`` is used to start the IDP app.
 
-**Case 1:** The method is not used to redirect from the RP app to the IDP app
+**Case 1:** `getCallingActivity()` is null
+
+`startActivityForResult()` was not used from the RP app to the IDP app
 
 In this case we can use the exact same method from above (``secureRedirection(uri: Uri)``).
 
-**Case 2:** The method is used to redirect from the RP app to the IDP app
+As noted for iOS, the RP native app may want to use a different redirect url to the web app if the user may start the flow from the system browser despite having the RP app installed.
```

Comment:
Should we really talk about web2app here? Because I think it would be better to write an additional section about web2app if this is an important use case.

Reply:
I'm not sure - it's more "if you're doing both". As people presumably already have the web-based flow working, I think the consideration is "when you add support for app2app, you should probably use a different redirect url than you're using on web". Does that make sense?

Reply:
Yes, this does make sense! Let us keep this paragraph there.